### DISCUSSION TITLE
Serbian translation files

### DISCUSSION THREAD

**Discussion by MilosPaunovic** [quasarframework/quasar#9797] [General - Components / Directives / etc] [answered]
I've noticed there is something a bit weird with Serbian translation files for both  [Q1](https://github.com/quasarframework/quasar/blob/v1/ui/lang/sr.js) and [Q2](https://github.com/quasarframework/quasar/blob/dev/ui/lang/sr.js) and wanted to check if it is meant to be like that or is it a mistake.

Majority of translation strings are written in Serbian latin alphabet, but there are a few ones in Cyrillic (which is also used in Serbian).  If it's made with intention - no problem, if it is a mistake, I'll be happy to submit a PR fix, just would like a confirmation first.

**Comment by rstoenescu** [comment, marked as answer]
Please do submit a PR. It should only be one alphabet used in the language pack. The one native Serbians use.
We rely 100% on community contributions to the (current -- or new!) language packs, so if there is a mistake we welcome fixes.
